Plugins/Flow.Launcher.Plugin.BrowserBookmark/Main.cs

```diff
@@ -85,6 +76,24 @@ public List<Result> Query(Query query)
             }
         }
 
+        private static bool ActionOpenUrl(ActionContext _, Bookmark c)
+        {
+            string profileArg = GetProfileArg(c);
+            context.API.OpenUrl(c.Url, true, profileArg);
+            return true;
+        }
+
+        private static string GetProfileArg(Bookmark c)
+        {
+            try
+            {
+                return c.Source.Split('(', ')')[1];
```

Comment:
what does this split mean?

Reply:
this is just a fancy way to extract the value between those chars

Reply:
Oh I see, so you are getting the content between the parenthesis?